### PR TITLE
[tests-only][full-ci]Add spaces tests on `apiAuthWebDav` suites

### LINKED ISSUE #3029: add spaces webdav for `apiAuthWebDav` suite
### Description
- use spaces WebDAV URL for `apiAuthWebDav` suite from `oc/core`
- includes:
  - webDavDELETEAuth.feature
  - webDavLOCKAuth.feature
  - webDavMKCOLAuth.feature
  - webDavMOVEAuth.feature
  - webdavMultiShashOC10Issue34365.feature (**`no need`**)
  - webDavPOSTAuth.feature
  - webDavPROPFINDAuth.feature
  - webDavPROPPATCHAuth.feature
  - webDavPUTAuth.feature
  - webDavPUTAuthOC10Issue39597.feature (**`no need`**)
  - webDavSpecialURLs.feature

### Related
- part of https://github.com/owncloud/ocis/issues/2854

### PR DESCRIPTION
**Pull request by amrita-shrestha**
## Description
use spaces DAV path in `apiAuthWebDav` suite

## Related Issue

- fixes https://github.com/owncloud/ocis/issues/3029

## How Has This Been Tested?

- Locally
- CI
- https://github.com/owncloud/ocis/pull/3030

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

File: tests/acceptance/features/apiAuthWebDav/webDavDELETEAuth.feature [hunk 1]
```diff
@@ -21,19 +21,39 @@ Feature: delete file/folder
...
       | /remote.php/dav/files/%username%/PARENT/parent.txt |
     Then the HTTP status code of responses on all endpoints should be "401"
 
+  @smokeTest @skipOnBruteForceProtection @issue-brute_force_protection-112 @skipOnOcV10 @personalSpace
+  Scenario: send DELETE requests to webDav endpoints as normal user with wrong password using the spaces WebDAV API
+    When user "Alice" requests these endpoints with "DELETE" including body "doesnotmatter" using password "invalid" about user "Alice"
+      | endpoint                                           |
+      | /remote.php/dav/spaces/%spaceid%/textfile0.txt     |
+      | /remote.php/dav/spaces/%spaceid%/PARENT            |
+      | /remote.php/dav/spaces/%spaceid%/PARENT/parent.txt |
```
Comment (kiranparajuli589) [inline tests/acceptance/features/apiAuthWebDav/webDavDELETEAuth.feature:34]:
```suggestion
      | /dav/spaces/%spaceid%/textfile0.txt     |
      | /dav/spaces/%spaceid%/PARENT            |
      | /dav/spaces/%spaceid%/PARENT/parent.txt |
```

**Review by kiranparajuli589 — DISMISSED** [review]
AFAIK, spaces DAV path does not contain `remote.php`. please change the URLs for every use case.

**Review by kiranparajuli589 — COMMENTED** [review]
looks good :+1:

File: tests/acceptance/features/apiAuthWebDav/webDavDELETEAuth.feature (hunk 1 above)
Comment (amrita-shrestha) [reply tests/acceptance/features/apiAuthWebDav/webDavDELETEAuth.feature:34]: `remote.php` is optional for all WebDAV variants (new, old, spaces) . In this tests scenario, the new and old variant is using `remote.php`. Just to make its structure similar I added remote.php in spaces tests too.

**Comment by phil-davis** [comment]
@kiranparajuli589 please give a "full green" review, or dismiss your previous red review.

**Review by kiranparajuli589 — APPROVED** [review]

**Review by phil-davis — APPROVED** [review]

**Pull request merged**